middlewares/error_pages_test.go

```diff
@@ -0,0 +1,66 @@
+package middlewares
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/codegangsta/negroni"
+	"github.com/containous/traefik/types"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestErrorPage(t *testing.T) {
+	testErrorPage := &types.ErrorPage{Backend: "error", Query: "/test", Status: []string{"500-501", "503-599"}}
+	testHandler, err := NewErrorPagesHandler(*testErrorPage, "http://example.com")
+
+	assert.Equal(t, nil, err, "Should be no error")
+	assert.Equal(t, testHandler.BackendURL, "http://example.com/test", "Should be equal")
+
+	recorder := httptest.NewRecorder()
+	req, err := http.NewRequest("GET", "http://example.com/test", nil)
```

Comment:
What if the build server has no internet access? I think we should use the `httptest` package here to spin up a dummy test server on the local loopback interface. [Here](https://golang.org/pkg/net/http/httptest/#Server) is an example. There is an existing test in Traefik where they do something similar [here](https://github.com/bparli/traefik/blob/64e8b31d4956e26886463a24094148542afd9717/healthcheck/healthcheck_test.go#L64).